content/blog/kubernetes-at-home-local-k8s-options.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: "Kubernetes at Home: Local k8s Options"
-featured: true
-weight: 3
+featured: false
```

Comment:
You could leave this one if you want and remove the Deployment/Packaging Pattern from Featured at this point...

Reply:
Cool, wasn't sure if we wanted to keep content mixed up, but those three blogs will be the most recent content

Reply:
Changed it around in the latest build